server/public/model/websocket_message_test.go

```diff
@@ -247,8 +247,11 @@ func BenchmarkEncodeJSON(b *testing.B) {
 
 	ev := message.PrecomputeJSON()
 
+	var seq int64
 	enc := json.NewEncoder(io.Discard)
 	for i := 0; i < b.N; i++ {
-		err = ev.Encode(enc)
+		ev = ev.SetSequence(seq)
+		err = ev.Encode(enc, io.Discard)
+		seq++
 	}
```

Comment:
Why do we need this here now? And why don't we use `int64(i)` directly?

Reply:
It just makes things a bit more realistic as that's what happens here: https://github.com/mattermost/mattermost/blob/ec4dc6b122a5a9b90d404adbd2f5150e6b5bc823/server/channels/app/platform/web_conn.go#L465-L467